Extensions/Firebase/C_firebasetools/D_analyticstools.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Firebase Tools Collection
+ * @fileoverview
+ * @author arthuro555
+ */
+
+/**
+ * Firebase Analytics Tools
+ * @namespace
+ */
+gdjs.evtTools.firebase.analytics = {};
+
+/**
+ * Logs an Event/Conversion for that user on the Analytics.
+ * @param {string} eventName The Event being triggered.
+ * @param {string} [eventData] Additional Data for the event.
```

Comment:
Update typing and comment to explain that a JS object or a JSON string are both supported.